include/core/rtos/EventFlags.hpp

```diff
@@ -0,0 +1,122 @@
+
+#ifndef _EVT_RTOS_EVENTFLAGS_
+#define _EVT_RTOS_EVENTFLAGS_
+
+#include <core/rtos/Initializable.hpp>
+
+namespace core::rtos {
+
+/**
+ * Class that wraps a ThreadX Event Flags Group. Event flags are essentially
+ * a 32-bit mask, with each bit representing a different flag.\n\n
```

Comment:
They format the comment better in the ide hover view. 
![image](https://github.com/user-attachments/assets/7832f27d-9713-43bc-b0ed-bed3d7895410)
You can see the gap between the line about extending Initializable. This makes it more readable because the super important information about the class is right there, and the less important, boilerplate information about how this functions as a class that extends Initializable is still there, but visually separated.